main.py

```diff
@@ -1,7 +1,36 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = ["George Klooney",
+              "Kianu Reaves",
+              "Scarlett Keegan",
+              "Will Smith",
+              "Jaden Smith",
+              "Scarlett Johansson"]
```

Comment:
You could place tuples with first name and last name inside of the list to use unpacking during the iteration. It will reduce the need to make a `split` operation on line 20. Refactor your code to achieve that.